xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3077,6 +3077,8 @@ platform.livetable.paginationResultsSingle=Result <span class="currentResultsNo"
 platform.livetable.paginationResultsMany=Results <span class="currentResultsNo">{0} - {1}</span> of <span class="totalResultsNo">{2}</span>
 platform.livetable.paginationResults=Results
 platform.livetable.paginationResultsOf=out of
+platform.livetable.false=No
+platform.livetable.true=Yes
```

Comment:
Hmm I see we also have:

    yes=Yes
    no=No

I don't know which one is best but this one might be better than the other one...

FTR I see other translations for yes/no... such as:

    core.widgets.confirmationBox.button.yes=Yes
    core.widgets.confirmationBox.button.no=No
    xe.officeimporter.openoffice.yes=Yes
    xe.officeimporter.openoffice.no=No

So the question is open: should we standardize on "yes"/"no" translation keys or should we introduce specific keys as Pascal has done above "platform.livetable.false"/"platform.livetable.true"?

I'm still hesitating on my side...